src/main/java/com/splunk/hecclient/HecAckPoller.java

```diff
@@ -184,6 +184,41 @@ public int getAckPollInterval() {
         return ackPollInterval;
     }
 
+    public void stickySessionHandler(HecChannel channel) {
```

Comment:
Include JavaDocs for new method

    /**
     * StickySessionHandler is .... It is used to .. 
     *
     * @param  channel  "channel is ...
     * @see          HecChannel
     * @since        1.1.0
     */